examples/dt/unified/delta/service-values.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderBackup:
+    customServiceConfigs: |
```

Comment:
All of the `customServiceConfigs` here are referenced elsewhere with the singular `customServiceConfig`, i.e. in `dt/unified/delta/kustomization.yaml`

Reply:
Thank you. Will modify them in the next commit.